lib/board_setup.rb

```diff
@@ -0,0 +1,60 @@
+require './lib/board'
+require './lib/ship'
+require './lib/game_message'
+
+class BoardSetup
+  attr_reader :board,
+              :computer_cruiser,
+              :computer_sub,
+              :player_cruiser,
+              :player_sub,
+              :game_messages
+  #this class should house setting up the board. Meaning placing boats.
+  #Iteration 4 = adding board dimensions
+
+  def initialize
+    @board = Board.new
+    @computer_cruiser = Ship.new("Cruiser", 3)
+    @computer_sub = Ship.new("Submarine", 2)
+    @player_cruiser = Ship.new("Cruiser", 3)
+    @player_sub = Ship.new("Submarine", 2)
+    @game_messages = GameMessage.new
+
+  end
+
+  def computer_select_cells(length)
+    randomized_cells = @board.cells.keys.shuffle
+    spaces = randomized_cells.shift(length)
+  end
+
+  def computer_select_cells_submarine
```

Comment:
Since we combined cruiser and submarine into one method I'm not sure we need this one anymore! What do you think?

Reply:
I agree, I removed it from the board_setup file and confirmed all tests still pass.